package.json

```diff
@@ -14,7 +14,7 @@
     "globby": "^10.0.2"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=15.0.0"
```

Comment:
Is there a particular reason we're aligning on Node `>=15.0.0`? Odd numbered node versions don't offer any kind of stability or support and generally just exist to allow community testing/feedback. I would recommend we align to an LTS version.

[Node release patterns](https://nodejs.org/en/about/releases/)

Reply:
This is probably us. We set our nvmrc to 15 a while back for support on another project and it has 'stuck'. I suspect we can move along to 16 now.
eg:
https://github.com/SU-SWS/adapt-dev-utils/blob/main/.nvmrc#L1

Reply:
NPM version has been updated to 16.